packages/react-native-editor/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-all.zip
```

Comment:
Warning (⚠️): I understand that you manually updated the `distributionUrl`. Actually, the correct way of updating Gradle is using not manually, but rather by using this Gradle wrapper command: `./gradlew wrapper --gradle-version=8.2.1 --distribution-type=all`

FYI: Try running this command a few times and until all, the `gradle-wrapper.jar`, `gradlew`
and `gradlew.bat` files get updated.

Reply:
Thanks for the call out 🙇 ! I originally changed it manually but ended up using the command `./gradlew wrapper --gradle-version=8.2.1 --distribution-type=all` after checking the Gradle documentation. Seems I forgot to push those changes when I created the PR, I'll do it now.

Reply:
Awesome, verified, thanks for that @fluiddot ! 🙇 ❤️ 🚀